readme.md

```diff
@@ -13,33 +13,30 @@ Utopia is an integrated design and development environment for React. It uses Re
 
 [Start the editor](https://utopia.app/project)
 
-# For contributors: Installing Utopia on your machine
+# For contributors only: Installing Utopia on your machine
 
-Utopia is browser-based. To run it locally, clone the repo, and then set up the server and webpack by following these instructions.
+(Please note: you shouldn't install Utopia on your machine in order to use it as a local dev environment: it's slower - sometimes very significantly so - and may still need connection to our servers. Also, you won't be able to edit projects in the file system on your machine if you install it locally.)
 
-## Prerequisites
+To install Utopia, you'll need to clone the repo, install the prerequisites, and then run the editor for the first time.
```

Comment:
maybe "contribute" is a bit too harsh, but I feel like we shouldn't even call this process "installing"

```suggestion
To contribute to Utopia, you'll need to clone the repo, install the prerequisites, and then run the editor for the first time.
```